lib/DXIL/DxilModule.cpp

```diff
@@ -1044,14 +1055,80 @@ static void RemoveResourcesWithUnusedSymbolsHelper(
   for (auto gv : eraseList) {
     gv->eraseFromParent();
   }
+  return modif;
+}
+} // namespace
+
+bool DxilModule::RemoveResourcesWithUnusedSymbols() {
+  bool modif = false;
+  modif |= RemoveResourcesWithUnusedSymbolsHelper(m_SRVs);
+  modif |= RemoveResourcesWithUnusedSymbolsHelper(m_UAVs);
+  modif |= RemoveResourcesWithUnusedSymbolsHelper(m_CBuffers);
+  modif |= RemoveResourcesWithUnusedSymbolsHelper(m_Samplers);
+  return modif;
+}
+
+bool DxilModule::RemoveEmptyBuffers() {
+
+  bool mod = false;
```

Comment:
Let's conform to variable naming standards and use `Changed`, as this is what's used for this purpose elsewhere.
```suggestion
  bool Changed = false;
```